### DISCUSSION TITLE
3d plots with Vizzu

### DISCUSSION THREAD

**Discussion by caleb-depotanalytics** [vizzuhq/ipyvizzu#302] [Q&A]
Hi!

Is there an ability to make 3D scatter plots in Vizzu similar to plotly.express.scatter_3d()?

Thanks

**Comment by simzer** [comment]
Hi!
Thanks, Good question.
There are no 3D plots in Vizzu for now. They are not on our roadmap either.
While. in my opinion, there are use-cases for 3D plots, they are usually hard to read. 
Our main angle on dataviz is to provide additional perspective to data through animation, which is kind of a complementary idea to adding a new spatial dimension. (an alternative for 3D scatter plot could be an animated scatter plot: https://jsfiddle.net/simzer/kp514hyj/26/)

**Reply by caleb-depotanalytics** [reply]
Thanks for the response! Can definitely understand why that would be tough to make work well in Vizzu framework. I actually like the animated scatter plot a lot. Thanks for sharing that example.
